packages/genesys-spark-components/src/components/beta/gux-time-zone-picker/gux-time-zone-picker.tsx

```diff
@@ -112,11 +130,25 @@ export class GuxTimeZonePickerBeta {
       if (!zone) {
         return;
       }
+
       timeZoneOptionsList.push(zone);
     });
-
     return timeZoneOptionsList.sort((a, b) => {
-      return a.displayTextName?.localeCompare(b.displayTextName) || 0;
+      if (
+        a.displayTextNameFormatted.startsWith('Etc/GMT') &&
+        !b.displayTextNameFormatted.startsWith('Etc/GMT')
+      ) {
+        return 1;
+      } else if (
+        !a.displayTextNameFormatted.startsWith('Etc/GMT') &&
+        b.displayTextNameFormatted.startsWith('Etc/GMT')
+      ) {
+        return -1;
+      } else {
+        return a.displayTextNameFormatted?.localeCompare(
+          b.displayTextNameFormatted
+        );
+      }
```

Comment:
This is sorting the list pushing all `time-zone-options` which start with `Etc/GMT` to the bottom of the list. If someone has a cleaner way let me know